app/react/projects/list/components/Locale.js

```diff
@@ -10,16 +12,24 @@ export default class Locale extends React.PureComponent {
   props: LocaleEntityType
 
   render() {
-    const { code, translationCount, translatedCount } = this.props;
+    const { projectId, code, translationCount, translatedCount, id } = this.props;
```

Comment:
I'm suggesting to rename `id` to `localeId`, just to be consistent with naming of different IDs in this component.